scss/_company_home.scss

```diff
@@ -0,0 +1,23 @@
+.company-home {
+  .company {
+    font-size: 1.5em;
+  }
+
+  .user {
+    font-size: 1em;
+  }
+
+  .rc-logo {
+    margin: 3em 0;
+    height: 10em;
+
+    img {
+      height: 100%;
+    }
```

Comment:
Going to separate the two issues from each other. I made changes to styling that make this obsolete.